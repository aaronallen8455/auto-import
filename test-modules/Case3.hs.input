module Case3 where

main :: IO ()
main = putStrLn $ Data.List.sort "hi"
