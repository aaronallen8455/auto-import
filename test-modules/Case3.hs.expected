module Case3 where

import qualified Data.List

main :: IO ()
main = putStrLn $ Data.List.sort "hi"
